debezium-core/src/main/java/io/debezium/transforms/debezium-timestamp-converter/src/main/java/org/telmate/SMT/DebeziumTimestampConverter.java

```diff
@@ -0,0 +1,695 @@
+
+/*
+Note: All the null values for any fields should be made to optional, if not while validating the value and schema, we would end up getting null value for field error.
+       Going to convert all null value columns into isoptional if they dont contains any data, so to avoid any kinds of (Conversion error: null value for field that is required and has no default value),
+       not just for nestedvalues, but also for struct fields.
+ */
+
+package org.telmate.SMT;
```

Comment:
Could you move the SMT into the `io.debezium.transforms` package instead (i.e. next to the existing ones)?